ios/CodePush/RCTConvert+CodePushInstallMode.m

```diff
@@ -1,5 +1,10 @@
 #import "CodePush.h"
+
+#if __has_include("RCTConvert.h")
```

Comment:
Should we swap the order, to do `#if __has_include(<React/RCTConvert.h>)` (that is, always check for the 0.40 case first)?  It shouldn't really matter, that that would make all the files consistent and seems slightly better.